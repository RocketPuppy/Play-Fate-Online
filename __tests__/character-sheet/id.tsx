import * as React from "react";
import * as renderer from "react-test-renderer";
import Id from "../../src/character-sheet/id";

test("renders all the pieces", () => {
  const render = renderer.create(<Id />);

  expect(render.toJSON()).toMatchSnapshot();
});

