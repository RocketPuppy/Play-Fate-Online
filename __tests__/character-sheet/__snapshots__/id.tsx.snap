// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Id
  </h2>
  <div>
    <label>
      <span>
        Name
      </span>
      <input
        onChange={[Function]}
        placeholder="Landon"
        type="text"
      />
    </label>
    <label>
      <span>
        Description
      </span>
      <textarea
        onChange={[Function]}
        placeholder="A rough-mannered swordsman covered in ragged clothing."
      />
    </label>
    <label>
      <span>
        Refresh
      </span>
      <input
        onChange={[Function]}
        placeholder="3"
        type="number"
      />
    </label>
  </div>
</div>
`;
